Adding GraphQL World

vning93 opened the pull request:
Adding [GraphQL World](https://graphql-world.com) & webinar series to Community page.

wincent commented on site/community/Community-Resources.md:
```diff
@@ -101,3 +103,4 @@ To explore other community-developed resources and content about GraphQL, take a
 - [Learn Apollo](https://www.learnapollo.com/): A hands-on tutorial for Apollo GraphQL Client (for React, ReactNative, Exponent, Angular, Vue & iOS)
 - [Learn Relay](https://www.learnrelay.org): A comprehensive introduction to Relay (includes video tutorials)
 - [Scaphold Community](https://scaphold.io/community/): Resources to help you launch a production app with GraphQL
+- [GraphQL World](https://graphql-world.com) - The fastest growing community of GraphQL developers
```
Can you switch the hyphen out for a colon please, to match the other entries here?

vning93 commented:
Just updated!

wincent commented:
Thanks @vning93. I just took another look at this and perhaps we should hold off on adding the link to the GraphQLWorld Twitter account until it has some more content (at this time it has published a total of two tweets). Can you back that change out and we'll land just the link to the website for now, we can add the Twitter link once the account has some regular activity under its belt.

Another question, do we have anything to backup the claim that GraphQL World is "[t]he fastest growing community"? It might be better to switch to something a little more descriptive of what it *is*; if it's mostly just the webinars, then perhaps the first link that you've added is enough. Otherwise, we should probably provide a better description. What do you think?

wincent commented:
Just a quick ping. It's been a month since the last activity on this PR. I'll close it shortly if it remains inactive (feel free to comment at any time and we can re-open it).

vning93 commented:
Sure thing!

I just removed the Twitter for now, and updated the description of GraphQL World as the "Global community of GraphQL developers and events".

Let me know if that works.

wincent commented:
Thanks @vning93. Looks good!
